easybuild/easyconfigs/g/goolf/goolf-1.5.16.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = "Toolchain"
+
+name = 'goolf'
+version = '1.5.16'
```

Comment:
@boegel Thanks, it save me some time. Next time I will ask for best naming scheme in advance.